### PR TITLE
Add global attribute api protocols

### PR DESCRIPTION
**Pull request by longquanzheng**

### CODE REVIEW

File: api-schema/xdb.yaml [hunk 1]
```diff
@@ -414,6 +427,105 @@ components:
...
+        alternativeTable:
+          type: string
+          description: if not empty, the global attribute is mapped to a different table name than the primary table. Must be used with alternativeTableForeignKeyColumn
+        alternativeTableForeignKeyColumn:
+          type: string
+          description: Must present when alternativeTable is not empty. It means the foreign key column to join with the primary key of the primary table
+    #        dbHint:
+    #          type: string
+    #          description: some additional info for how to use the dbColumn and dbQueryValue     
+    GlobalAttributeRequest:
```
Comment (zklgame) [inline api-schema/xdb.yaml:488]: Should we add the verb on so that making the name to `LoadGlobalAttributeRequest`?

Comment (zklgame) [reply api-schema/xdb.yaml:488]: And maybe make it plural -> `LoadGlobalAttributesRequest `

File: api-schema/xdb.yaml [hunk 2]
```diff
@@ -414,6 +427,105 @@ components:
...
+      type: object
+      description: the response for loading global attributes
+      properties:
+        attributes:
+          type: array
+          items:
+            $ref: '#/components/schemas/GlobalAttributeValue'
+        error:
+          $ref: '#/components/schemas/GlobalAttributeLoadingError'
+    TableReadLocking:
```
Comment (zklgame) [inline api-schema/xdb.yaml:513]: Add `Policy` on to make the naming more consistent?

**Review by zklgame — APPROVED** [review]

File: api-schema/xdb.yaml (hunk 1 above)
Comment (longquanzheng) [reply api-schema/xdb.yaml:488]: Yeah good idea

**Pull request merged**